src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/MultivariateGaussian.java

```diff
@@ -22,12 +22,14 @@
     public double hyperParameter_lambda;
     private double cachedDenomLog10;
     private Matrix cachedSigmaInverse;
-    final private ExpandingArrayList<Double> pVarInGaussian;
+    final private double[] pVarInGaussian;
```

Comment:
Previous implementation had an `ExpandingArrayList`, now you have a fixed array -- does the fixed array need to expand beyond `numVariants`?